doc/integrator/create_application.rst.tmpl

```diff
@@ -113,8 +113,45 @@ Go to your new project:
 
 .. prompt:: bash
 
-    cd ${GEOMAPFISH_PROJECT}
+    cd ${DOLLAR}{GEOMAPFISH_PROJECT}
 
+Simple application
+------------------
+
+A simple application is an application that you can't add come code on it (Python or JavaScript).
```

Comment:
A simple application is an application for which no custom code is needed (Python or JavaScript).